src/components/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -125,15 +125,19 @@
               </span>
 
               <!-- More Actions Menu -->
-              <span class="more-actions mr-4">
-                <v-menu offset-y>
+              <span :class="`more-actions-${index}`" class="mr-4">
```

Comment:
This dynamic class is to be able to attach directly to it in the v-menu below (otherwise items 2..x attach to the first item :( ).